src/gui/tracks/TrackLabelButton.cpp

```diff
@@ -60,8 +63,8 @@ TrackLabelButton::TrackLabelButton( TrackView * _tv, QWidget * _parent ) :
 	else
 	{
 		setFixedSize( 160, 29 );
-		m_renameLineEdit->move( 30, ( height() / 2 ) - ( m_renameLineEdit->sizeHint().height() / 2 ) );
-		m_renameLineEdit->setFixedWidth( width() - 33 );
+		m_renameLineEdit->move(25, ( height() / 2  - m_renameLineEdit->sizeHint().height() / 2 ) + 1);
```

Comment:
```suggestion
		m_renameLineEdit->move(25, (height() / 2  - m_renameLineEdit->sizeHint().height() / 2) + 1);
```

Reply:
That was a part of my initial review. The OP forgot to remove those spaces.

Reply:
I didn't double check the diff either.

Reply:
Fixed with commit c60b5fed5a5.